src/distances.jl

```diff
@@ -3,13 +3,21 @@
 # ------------------------------------------------------------------
 
 """
-    Ellipsoidal(semiaxes, angles)
+    Ellipsoidal(semiaxes, angles; convention=:TaitBryanExtr)
 
 A distance defined by an ellipsoid with given `semiaxes` and rotation `angles`.
+Default angle convention is Tait-Bryan ZXY, extrinsic right-handed rotation
+Other conventions available: :TaitBryanIntr, :EulerExtr, :EulerIntr,
+                             :GSLIB, :Leapfrog, :Datamine
+* :EulerExtr, :EulerIntr and :Datamine assumes ZXZ rotation sequence
 
 - For 2D ellipsoids, there are two semiaxes and one rotation angle.
 - For 3D ellipsoids, there are three semiaxes and three rotation angles.
 
+    Ellipsoidal(semiaxes, quaternion)
+
+Alternatively, give the `semiaxes` and the orientation as a `quaternion`
+
```

Comment:
I am also curious if we should even include this quaternion constructor. It seems that the other constructor will be super useful already and will cover all cases that one could cover with quaternions, agree? What do you think?

Reply:
I guess the only cases not covered are strange rotations such as :YZX or something like that. About that. Do you think it'd be advisable to export RotationRule() and give the option to inform custom rules or let restrict it inside the code?

Reply:
Let's keep it inside the code for now, I think most people won't bother creating their own rules, and will instead pick one of the available ones.